server/models/queries.js

```diff
@@ -14,6 +14,12 @@ WHERE landlords.overall_rating != 'NaN'
 ORDER BY overall_rating DESC 
 LIMIT 4;`;
 
+queries.addLandlord = `INSERT INTO landlords (is_company, user_id)
+VALUES ($1, $2)
+RETURNING _id`
+
+queries.addLandlordId = `UPDATE users SET landlord_id = $1 WHERE _id = $2 RETURNING *`
```

Comment:
if `landlord_id` is removed as a foreign key from `users`, this query is not necessary

Reply:
deleted!